pageserver/src/tenant.rs

```diff
@@ -4670,8 +4670,8 @@ impl TenantShard {
                 // Look up parent's PITR cutoff to update the child's knowledge of whether it is within parent's PITR
                 if let Some(ancestor_id) = timeline.get_ancestor_timeline_id() {
                     if let Some(ancestor_gc_cutoffs) = gc_cutoffs.get(&ancestor_id) {
-                        target.within_ancestor_pitr =
-                            timeline.get_ancestor_lsn() >= ancestor_gc_cutoffs.time;
+                        target.within_ancestor_pitr = timeline.get_ancestor_lsn()
```

Comment:
Nit - might be a bit more idiomatic to do `Some(timeline.get_ancestor_lsn()) >= ancestor_gc_cutoffs.time` to avoid reader having to know that the Lsn::Default() value compares less than other values.